connect-button/src/test/java/com/ifttt/connect/ui/TestActivity.java

```diff
@@ -15,6 +18,12 @@
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        final Configuration config = new Configuration.Builder()
```

Comment:
Why do we need this?

Reply:
Was related to WorkManager testing, will remove since we aren't testing it.